rotkehlchen/tasks/manager.py

```diff
@@ -227,7 +227,7 @@ def _prepare_cryptocompare_queries(self) -> None:
 
         with self.database.conn.read_ctx() as cursor:
             assets = self.database.query_owned_assets(cursor)
-            main_currency = self.database.get_setting(cursor=cursor, name='main_currency')
+            main_currency = self.database.get_setting(cursor=cursor, name='main_currency').resolve_to_asset_with_oracles()  # noqa: E501
```

Comment:
So wait is it asset with oracles or not? In the other part of the code you changed it to be `Asset`. It makes sense to be `AssetWithOracles`, so can you change it back please?